README.md

```diff
@@ -138,6 +138,7 @@ eval "$shellHook"
 
 - [prettier](https://prettier.io)
 - [clang-format](https://clang.llvm.org/docs/ClangFormat.html)
+- [hadolint](https://github.com/hadolint/hadolint)
```

Comment:
I'm not sure hadolint a _formatter_, it might be worth creating a `Docker` section above the `Other Formatters` heading